includes/Webhook/Hooks/OrderStatusWebhook.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\WooCommerce\includes\Webhooks\Hooks;
+
+defined('ABSPATH') or die();
+
+use Exception;
+use MyParcelNL\Sdk\src\Collection\Fulfilment\OrderCollection;
+use MyParcelNL\Sdk\src\Services\Web\Webhook\OrderStatusChangeWebhookWebService;
+use MyParcelNL\WooCommerce\includes\Concerns\HasApiKey;
+use MyParcelNL\WooCommerce\includes\Concerns\HasInstance;
+use MyParcelNL\WooCommerce\includes\Webhook\Service\WebhookSubscriptionService;
+use WCMP_Export_Consignments;
+use WCMP_Log;
+use WCMP_Settings_Data;
+use WCMYPA_Settings;
+use WP_REST_Request;
+use WPO\WC\MyParcel\Compatibility\WC_Core;
+
+class OrderStatusWebhook
+{
+    use HasApiKey;
+    use HasInstance;
+
+    /**
+     * 2 : Package shipment barcode printed
+     * 12: Letter shipment barcode printed
+     * 14: Digital stamp barcode printed
+     */
+    public const COMPLETED_SHIPMENT_STATUSES = [
+        2,
+        12,
+        14,
+    ];
+
+    /**
+     * @throws \Exception
+     */
+    public function __construct()
+    {
+        $changeOrderStatusAfter = WCMP_Export_Consignments::getSetting(WCMYPA_Settings::SETTING_CHANGE_ORDER_STATUS_AFTER);
```

Comment:
dit ook in een eigen functie

Reply:
Misschien kun je ook even kritisch kijken naar de overeenkomsten tussen de classes in `Webhook/Hooks`, die kunnen waarschijnlijk gewoon allebei dezelfde parent class gebruiken waardoor er wat code weg kan